plugins/inspect/patterns_inspector.rb

```diff
@@ -37,9 +42,9 @@ def summary(description)
 
   private
 
-  def inspect_with_zypper(system, description)
+  def inspect_with_zypper
```

Comment:
Assignment Branch Condition size for inspect_with_zypper is too high. [24.52/15]